src/headers/tfm.h

```diff
@@ -9,11 +9,26 @@
 #ifndef TFM_H_
 #define TFM_H_
 
+#if TFM_NO_STDLIB
+/* do nothing */
```

Comment:
This worries me a bit, 'cause it allows anyone to defined `-DTOM_NO_STDLIB=1` regardless of compiler, and when looking through the code, this feature makes heavy use of GCC features.

People might want to build without having to link with stdlib for other reasons than webasm...

It's possible that this should simply be documented with fierce warnings not to try this with other compilers than GCC or derivatives thereof (like clang)